### ISSUE TITLE
Care if i add this to goblin? 

### ISSUE THREAD

**Issue by mortdeus** [rjkroege/edwood#275]
So, i dont know if you guys have ever heard about Goblin or somebody named Uriel (who used to be a big part of our community way back in the earliest days of Go's development and unfortunately passed away), but essentially the goal of that project was to reimplement a pure go version of the plan 9 userland, which is a key component to building a virtual host OS (similar to inferno) using goroot as the root directory.

Anyways, acme is a key part to that, and I need a rewritten version in Go and not just a straight 
1:1 transpiled version using the existing code base due to the crappy licensing were stuck with otherwise. 

plan 9 as i understand it, is currently licensed as GPLv2 (now i believe anyways) and LucentPL which the FSF rejects as a GPLv2 compatible license. Or something like that, it's been awhile since i looked into it, but my point is that we want to be able to use Go's BSD license.

So basically what you guys are doing right now seems to be the right idea. And maybe its a good time to start working on goblin again?

**Comment by mortdeus** [comment]
heres the link. 

http://repo.cat-v.org/goblin/

Uriel created cat-v.org and made a lot of contributions to wmii, so its kinda in his honor that i took on this project but it turned out to be more i could handle at the time.

**Comment by rjkroege** [comment]
@mortdeus I think it would be cool if you included Edwood in Goblin. It's up to you if the Edwood license meets your needs. Do you have a list of tools that you'd include. I know that there's a `mk` and `awk` clone written in Go. And some cool shells. But I'm not clear what the actual bug is here? Maybe you could clarify?

**Comment by rjkroege** [comment]
I'm going to close this as non-actionable. I'm good with Goblin including Edwood.

**Comment by mortdeus** [comment]
Thanks. One thing I recommend, is to change the license so that instead of referencing the Lucent code as being under the Lucent PL license, to instead refer to the plan 9 code that Lucent relicensed under the GPL license because the Lucent license isn't legally cross compatible with GPL licensed code.
